Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(platforms: ['windows'])
\ No newline at end of file
+buildPlugin(configurations: buildPlugin.recommendedConfigurations())
```

Comment:
Are we sure we want to test it in linux as well?

Reply:
probably not indeed given the previous settings indeed.